app/src/main/res/layout/main_fragment.xml

```diff
@@ -8,78 +8,76 @@
+                    android:text="Schedule"
+                    android:textSize="9dp" />
 
-        <Button
-            android:id="@+id/btnCatagories"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="40dp"
-            android:layout_marginLeft="40dp"
-            android:text="Catagories"
-            android:textSize="9dp"
-            app:layout_constraintStart_toEndOf="@+id/btnShoppingList"
-            tools:layout_editor_absoluteY="105dp" />
+                <Button
+                    android:id="@+id/btnShoppingList"
+                    android:layout_width="100dp"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="10dp"
+                    android:text="Shopping List"
+                    android:textSize="9dp" />
 
-        <Button
-            android:id="@+id/btnSchedule"
-            android:layout_width="100dp"
+        </LinearLayout>
+
+        <AutoCompleteTextView
+            android:id="@+id/rcpSearch"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginStart="40dp"
-            android:layout_marginLeft="40dp"
-            android:layout_marginEnd="10dp"
-            android:layout_marginRight="10dp"
-            android:text="Schedule"
-            android:textSize="9dp"
+
+            android:gravity="center_horizontal"
+
+            android:hint="Select a recipe"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.166"
-            app:layout_constraintStart_toEndOf="@+id/btnCatagories"
-            tools:layout_editor_absoluteY="105dp" />
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/linearLayout" />
 
         <TextView
             android:id="@+id/txtShowing"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="10dp"
-            android:layout_marginBottom="10dp"
-            android:text="Showing:    Recipes"
-            app:layout_constraintBottom_toTopOf="@+id/lstRecipes"
-            app:layout_constraintTop_toBottomOf="@+id/btnCatagories"
             android:gravity="center_horizontal"
-             />
+            android:text="Showing:    Recipes"
+            app:layout_constraintBottom_toBottomOf="@+id/rcpSearch"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/rcpSearch" />
+
+        <ListView
+            android:id="@+id/lstRecipes"
+            android:layout_width="405dp"
+            android:layout_height="629dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/txtShowing">
+
+        </ListView>
```

Comment:
I made sure your list view was constrained to the bottom of the screen. In the future, you may want to change your list view to a recycler view if you would like to add more advanced functionality.